samesite.vcl

```diff
@@ -0,0 +1,37 @@
+sub samesite {
+        # See list here:
+        # https://www.chromium.org/updates/same-site/incompatible-clients
+        unset req.http.X-UA-SameSiteNone;
+        set req.http.X-UA-SameSiteNone = "supported";
+
+        # Versions of Chrome from Chrome 51 to Chrome 66 (inclusive on both ends). These Chrome versions will reject a cookie with `SameSite=None`
+        if (req.http.user-agent ~ "Chrom(e|ium)" &&
```

Comment:
no need to test the header existence

Reply:
Hi Guillaume, are you talking about the VCL or VTC part ?

Reply:
hum, I must have misread that code, I was sure it was
```
if (req.http.user-agent && req.http.user-agent ~ "Chrom(e|ium)" &&
```

nevermind me...